scrabble-score.json

```diff
@@ -0,0 +1,64 @@
+{
+  "cases": [
+    {
+      "description": "Short input",
+      "input": "a",
+      "expected": 1,
+      "msg": "No, 'a' scores 1 point"
```

Comment:
Let's ditch the `msg` in this exercise; it's not adding any information beyond the expected.

We needed it for one exercise where the expected was just a boolean, and the assertions ended up being confusing.